pipeline-modules/deployment-service/aws/istio-app.yaml

```diff
@@ -36,10 +36,38 @@ inputs:
       title: AWS Role ARN
       description: AWS Role ARN to be user
       type: string
+    enable_istio_access_log:
+      title: Istio-proxy Access Log
+      description: To enable Istio access logging
+      type: boolean
+      default: true  
+    job_type:
+      title: Job Type
+      description: This is to deploy or undeploy application
+      type: string
+      default: deploy
```

Comment:
This can be removed as it already comes in via Code.

Reply:
job_type input remove as suggested